ReSharper.FSharp/src/FSharp.Psi.Services/src/Daemon/Highlightings/Errors.xml

```diff
@@ -326,4 +330,11 @@
     <Range>getIndexerDotRange indexerExpr</Range>
     <QuickFix>RemoveIndexerDotFix</QuickFix>
   </Warning>
+
+  <Warning name="NestedWithCanBeSimplified" configurableSeverity="NestedWithCanBeSimplified">
```

Comment:
I'd probably vote for something like 'nested record' or 'nested record update' naming instead of 'nested with', as `with` have other usages in the language, and this name may be less clear without the context.